Studies/source/LegacyConvert.cxx

```diff
@@ -0,0 +1,129 @@
+/*! Study for Mva Training
```

Comment:
I think this file shouldn't be part of the commit. Keep it in your private repo.